catroid/src/org/catrobat/catroid/content/Project.java

```diff
@@ -424,4 +421,33 @@ public boolean containsSprite(Sprite selectedSprite) {
 		}
 		return false;
 	}
+
+	public void addSprite(int spriteIndex, Sprite sprite) {
+		spriteList.add(spriteIndex, sprite);
+	}
+
+	public void refreshSpriteReferences() {
+		Project project = ProjectManager.getInstance().getCurrentProject();
+		for (Brick brick : project.getAllBricks()) {
+			if (!(brick instanceof PointToBrick)) {
+				continue;
+			}
+			PointToBrick pointToBrick = (PointToBrick) brick;
+			Sprite pointedSprite = pointToBrick.getPointedObject();
+			if (pointedSprite == null) {
+				continue;
+			}
+
+			Sprite newSprite = getSpriteBySpriteName(pointToBrick.getPointedObject());
```

Comment:
Ok I understand. The groups should not appear in the 

The group should not appear in the dropdown list of sprites in the PointToBrick, i guess?
https://www.dropbox.com/s/ddhtq0gw292xf57/2016-08-10%2017.26.07.png?dl=0